deploy-image/.aea/cli_config.yaml

```diff
@@ -1,2 +0,0 @@
-auth_token: <your-auth-token>
```

Comment:
Is it needed @marcofavorito ?

Reply:
I can see the `Dockerfile` copies it into the image, so at least it is used. However, I also think we might not need it, as the CLI tool will ask the user to authenticate if it can't find the configuration file.